app/views/users/index.html.erb

```diff
@@ -1 +1,11 @@
-<h1>Here is a list of all users</h1>
\ No newline at end of file
+<section class= "all">
+  <% @users.each do |user| %>
+    <div class="user_container">
+   <img class="pic" src="<%= user.photo %>" alt="profile_pic">
+   <div class=>
+   <%= link_to user.name, user_path(user)%>
+  <p class="post-count">Number of posts: <%= user.posts_counter || 0 %></p>
+</div>
```

Comment:
- [ ] Kindly as requested by the previous reviewer, the user index view page should match the given [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_root_users.png) However your design does not match the wireframe👇🏻 Kindly make sure that the index view page matches the given the user index view page should match the given [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_root_users.png).

Your Implementation | Original Design
:---:|:---:
![image](https://user-images.githubusercontent.com/79058364/233747379-d9acb96b-c89b-415a-abee-5d4deda33a1c.png) | ![image](https://user-images.githubusercontent.com/79058364/233747438-62834c84-a19a-490b-9201-2997b7a318cd.png)


- Do the same for your User show page.

![image](https://user-images.githubusercontent.com/79058364/233748112-c699af57-868d-4204-9c0d-520a26bb3a3c.png)